packages/search-spa/set-env.ts

```diff
@@ -7,7 +7,7 @@ const targetPath = './src/environments/environment.ts';
 require('dotenv').config();
 
 const envConfigFile = `export const environment = {
-   API_URL: process.env.API_URL,
+   API_URL: '${process.env.API_URL || ''}',
```

Comment:
```suggestion
   API_URL: process.env.API_URL || '',
```

Reply:
@ghanlohar It's already in a string template, so this won't work.

Reply:
This actually breaks the env file generation.

Reply:
Yeah I see. Looks good. Thanks.